lib/src/model.dart

```diff
@@ -1258,7 +1258,17 @@ class Parameter extends ModelElement {
 
   @override
   String get _href {
-    throw new UnsupportedError('parameters do not have hrefs');
+    var p = _parameter.enclosingElement;
```

Comment:
Why does a parameter need _href ? Isn't _href only used when we're generating a page for it?